pkg/cryptoutil/keystore_message.go

```diff
@@ -330,47 +335,49 @@ type computedKey struct {
 }
 
 func (m *MessageKeystore) putPrecomputedKeys(ctx context.Context, groupPK, device crypto.PubKey, preComputedKeys ...computedKey) error {
-	if m == nil || len(preComputedKeys) == 0 {
+	if m == nil {
```

Comment:
So `len(preComputedKeys) == 0` doesn't return an error any more, is it what you expect for?

Reply:
Yes that's the point of this fix

Reply:
Yes that's the point of the fix